weeder.dhall

```diff
@@ -2,6 +2,7 @@
   [ "^Main.main\$"
   , "^Paths_"
   , "^Ormolu.Terminal.QualifiedDo.>>\$" -- https://github.com/ocharles/weeder/issues/112
+  , "^Ormolu.Fixity.Parser.pPackageName"
   ]
-, type-class-roots = False
+, type-class-roots = True
```

Comment:
Necessary because some functions are only used to define `Arbitrary` instances (in tests).